pkg/util/errors/errors.go

```diff
@@ -26,11 +26,11 @@ var (
 )
 
 func unwrapNetHTTPError(err error) (bool, error) {
-	if !strings.HasPrefix(err.Error(), netHTTPErrorPrefix) {
+	if !strings.Contains(err.Error(), netHTTPErrorPrefix) {
 		return false, err
 	}
 	for sysMsg, errno := range str2Syscall {
-		if strings.Contains(err.Error()[netHTTPErrorPrefixLen:], sysMsg) {
+		if strings.Contains(err.Error(), sysMsg) {
```

Comment:
can you also remove the `netHTTPErrorPrefixLen` at the top of the file?

Reply:
sure, forgot about that